src/SFA.DAS.RoatpGateway.Web/Views/Gateway/Pages/_GatewayClarificationDetails.cshtml

```diff
@@ -15,23 +15,23 @@
             Date of check
         </dt>
         <dd class="govuk-summary-list__value">
-            @Model.OutcomeMadeOn.ToSfaShortDateString()
+            @Model.ClarificationDate.ToSfaShortDateString()
         </dd>
     </div>
     <div class="govuk-summary-list__row">
         <dt class="govuk-summary-list__key">
             Comments
         </dt>
         <dd class="govuk-summary-list__value">
-            @Model.Comments
+            @Model.ClarificationComments
         </dd>
     </div>
     <div class="govuk-summary-list__row">
         <dt class="govuk-summary-list__key">
             Outcome made by
         </dt>
         <dd class="govuk-summary-list__value">
-            @Model.OutcomeMadeBy
+            @Model.ClarificationBy
```

Comment:
I'm not sure how well this will play with 2033 (withdrawal) as it uses OutcomeMadeBy